src/SamplesApp/UITests.Shared/Windows_UI_Xaml/DragAndDrop/DragDrop_TestPage.xaml.cs

```diff
@@ -0,0 +1,393 @@
+		private Brush DragLeaveBrush = new SolidColorBrush(Colors.LightSalmon);
+		private Brush DragOverBrush = new SolidColorBrush(Colors.LightBlue);
+		private Brush DropBrush = new SolidColorBrush(Colors.LightPink);
+		private Brush DropCompletedBrush = new SolidColorBrush(Colors.Tomato);
+
+		public DragDrop_TestPage()
+		{
+			this.InitializeComponent();
+			this.DataContext = this;
+		}
+
+		/// <summary>
+		/// Gets or sets a value indicating whether duplicate drag drop events will be shown right after one another.
+		/// </summary>
+		public bool ShowDuplicateEvents { get; set; } = false;
+
+		/// <summary>
+		/// Gets the list of past drag and drop events.
+		/// </summary>
+		public ObservableCollection<EventDetails> DragDropEvents { get; } = new ObservableCollection<EventDetails>();
+
+		private void AddToDragDropEvents(EventDetails eventDetails)
+		{
+			if (this.ShowDuplicateEvents ||
+				this.DragDropEvents.Count < 1 ||
+				eventDetails.Equals(this.DragDropEvents[0]) == false)
+			{
+				this.DragDropEvents.Insert(0, eventDetails);
+				while (this.DragDropEvents.Count > MaxDragDropEventsListCount)
+				{
+					this.DragDropEvents.RemoveAt(this.DragDropEvents.Count - 1);
+				}
+			}
+
+			return;
+		}
+
+		/// <summary>
+		/// Displays the contents of the given <see cref="global::Windows.UI.Xaml.DragEventArgs"/> for any drag/drop event.
+		/// </summary>
+		/// <param name="eventName">The name of the event to display.</param>
+		/// <param name="args">The event arguments to display.</param>
+		private void ShowDragDropDetails(string eventName, global::Windows.UI.Xaml.DragEventArgs args)
+		{
+			// Determine the standard formats
+			var standardDataFormats = string.Empty;
+			var sep = Environment.NewLine;
+			if (args.DataView.Contains(StandardDataFormats.ApplicationLink)) { standardDataFormats += nameof(StandardDataFormats.ApplicationLink) + sep; }
+			if (args.DataView.Contains(StandardDataFormats.Bitmap)) { standardDataFormats += nameof(StandardDataFormats.Bitmap) + sep; }
+			if (args.DataView.Contains(StandardDataFormats.Html)) { standardDataFormats += nameof(StandardDataFormats.Html) + sep; }
+			if (args.DataView.Contains(StandardDataFormats.Rtf)) { standardDataFormats += nameof(StandardDataFormats.Rtf) + sep; }
+			if (args.DataView.Contains(StandardDataFormats.StorageItems)) { standardDataFormats += nameof(StandardDataFormats.StorageItems) + sep; }
+			if (args.DataView.Contains(StandardDataFormats.Text)) { standardDataFormats += nameof(StandardDataFormats.Text) + sep; }
+			if (args.DataView.Contains(StandardDataFormats.UserActivityJsonArray)) { standardDataFormats += nameof(StandardDataFormats.UserActivityJsonArray) + sep; }
+			if (args.DataView.Contains(StandardDataFormats.Uri)) { standardDataFormats += nameof(StandardDataFormats.Uri) + sep; }
+			if (args.DataView.Contains(StandardDataFormats.WebLink)) { standardDataFormats += nameof(StandardDataFormats.WebLink) + sep; }
+
+			if (standardDataFormats.EndsWith(sep))
```

Comment:
Codacy found an issue: [Change this call to 'standardDataFormats.EndsWith' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6334034)